source/GreenEnergyHub.Charges/source/GreenEnergyHub.Charges.Application/ChargeInformations/Factories/ChargeInformationCreatedEventFactory.cs

```diff
@@ -12,26 +12,26 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-using GreenEnergyHub.Charges.Application.Charges.Acknowledgement;
+using GreenEnergyHub.Charges.Application.ChargeInformations.Acknowledgement;
 using GreenEnergyHub.Charges.Core.Currency;
-using GreenEnergyHub.Charges.Domain.Charges;
+using GreenEnergyHub.Charges.Domain.ChargeInformations;
 using GreenEnergyHub.Charges.Domain.Dtos.ChargeCommands;
 
-namespace GreenEnergyHub.Charges.Application.Charges.Factories
+namespace GreenEnergyHub.Charges.Application.ChargeInformations.Factories
 {
-    public class ChargeCreatedEventFactory : IChargeCreatedEventFactory
+    public class ChargeInformationCreatedEventFactory : IChargeInformationCreatedEventFactory
     {
         private readonly CurrencyConfigurationIso4217 _currencyConfigurationIso4217;
 
-        public ChargeCreatedEventFactory(CurrencyConfigurationIso4217 currencyConfigurationIso4217)
+        public ChargeInformationCreatedEventFactory(CurrencyConfigurationIso4217 currencyConfigurationIso4217)
         {
             _currencyConfigurationIso4217 = currencyConfigurationIso4217;
         }
 
-        public ChargeCreatedEvent Create(ChargeOperationDto chargeOperationDto)
+        public ChargeInformationCreatedEvent Create(ChargeOperationDto chargeOperationDto)
         {
-            return new ChargeCreatedEvent(
-                chargeOperationDto.ChargeId,
+            return new ChargeInformationCreatedEvent(
+                chargeOperationDto.ChargeInformationId,
```

Comment:
I prefer to keep the prop name `ChargeId` on chargeOperationDto. 
Adding information to me seems confusing, when we are talking about the actual charge id.

```suggestion
                chargeOperationDto.ChargeInformationId,
```

Reply:
ok